portfolio/src/main/webapp/script.js

```diff
@@ -13,29 +13,31 @@
 // limitations under the License.
 
 let i = 0;
-let facts = [
-  "Digital Arts and Sciences Engineer ",
-  "Full-Stack Developer ",
-  "Scrum Master ",
-  "Google SPS participant! >:) "
-];
-let txt = "I'm a " + facts[0];
+let txt = "I'm a ";
 let speed = 75;
 let backspace = false;
 let blinkCursor = true;
 
 window.onload = function() {
+  fetchText(0);
   typeFact();
-  i = 0;
-  txt = "I'm a " + facts[0];
-  backspace = false;
 };
 
+async function fetchText(index) {
+    const responseFromServer = await fetch('/servlet');
+    let facts = await responseFromServer.json();
+    if(index >= 0) {
+        txt = "I'm a " + facts[index];
+    } else {
+        txt = "I'm a " + facts[Math.floor(Math.random() * facts.length)];
+    }
+}
+
 function typeButton() {
```

Comment:
You might consider adding a comment above this function explaining what it does?